tests/parser/features/test_init.py

```diff
@@ -53,3 +53,53 @@ def baz() -> uint8:
 
     n = 256
     assert_compile_failed(lambda: get_contract(code, n))
+
+
+# issue 3292
```

Comment:
I think it's good practice to add the links if you want to reference the issues

Reply:
i'm not a huge fan of including links in code since they are kind of cluttering (since they are long), links can get broken, the link could be misspelled and accidentally takes you to a phishing site (:smiling_face_with_tear: ), etc. also i have already been in the habit of including issue numbers in this format, and it seems to be working.

if it's ambiguity you are concerned about (like in the future we move to bitbucket or something) - maybe i can change to `GH issue 3292`? this is also machine readable and greppable if we want to add some kind of tooling for that in the future.

Reply:
As discussed via offline discussion, we decided for `GH issue X`.

Reply:
discussed offline with @pcaversaccio, `GH issue X` it is

Reply:
done in bbfaf2ab6c17abec9660cfeac76e5db1796da398